Orchestration/OrchestrationService/ModuleConfigurationDeployment.ps1

```diff
@@ -23,7 +23,14 @@
     $TearDownValidationResourceGroup,
     [Parameter(Mandatory=$false)]
     [switch]
-    $TearDownEnvironment)
+    $TearDownEnvironment,
+    [Parameter(ParameterSetName='Transcript',Mandatory=$false)]
+    [switch]
+    $GenerateTranscript,
+    [Parameter(ParameterSetName='Transcript',Mandatory=$true)]
+    [string]
+    $TranscriptPath
```

Comment:
Is this expecting you to pass a path? If yes, we might want to offer a default path if none is provided.

Reply:
No changes made as per discussion.